storage/google/cloud/storage/bucket.py

```diff
@@ -345,7 +345,22 @@ def user_project(self):
         """
         return self._user_project
 
-    def blob(self, blob_name, chunk_size=None, encryption_key=None, kms_key_name=None):
+    @property
+    def _query_params(self):
+        """Default query parameters."""
+        params = {}
+        if self.user_project is not None:
+            params["userProject"] = self.user_project
+        return params
```

Comment:
Please add explicit test coverage for the branches in this property.

Reply:
Per above, let's just delete this version of the property, which duplicates the one in the base class.